aii-core/src/main/perl/Shellfe.pm

```diff
@@ -229,19 +231,23 @@ sub app_options
          HELP    => 'where to store lock files',
          DEFAULT => "/var/lock/quattor" },
 
+       { NAME    => 'parallel=i',
+         HELP    => 'run commands on hosts in parallel ',
+         DEFAULT => 0 },
+
+       { NAME    => 'logpid',
+         HELP    => "Add process ID to the log messages (disabled by default, unless parallel is > 1)",
+         DEFAULT => undef },
+
          # Options for osinstall plug-ins
        { NAME    => 'osinstalldir=s',
          HELP    => 'Directory where Kickstart files will be installed',
          DEFAULT => '/osinstall/ks' },
 
          # Options for DISCOVERY plug-ins
-         { NAME => 'dhcpconf=s',
-           HELP => 'name of dhcp configuration file',
-           DEFAULT => '/etc/dhcpd.conf' },
```

Comment:
@jrha it's used by `aii-dhcp` or `AII::DHCP`, this is shellfe.
@jouvin has a valid concern, but we also have this entry, and things still seem to work.

what @kwaegema removed is an option that suggested you could change the dhcpd.conf location via aii-shellfe `.conf` or commanline ; but you can't. aii-dhcp/AII::DHCP however picks up this config file.